src/globus_cli/commands/timer/create.py

```diff
@@ -0,0 +1,288 @@
+@click.option(
+    "--name",
+    required=True,
+    type=str,
+    help="Name to identify this job in the timer service (not necessarily unique)",
+)
+@click.option(
+    "--start",
+    required=False,
+    type=click.DateTime(formats=DATETIME_FORMATS),
+    help=START_HELP,
+)
+@click.option(
+    "--interval",
+    required=False,
+    type=str,
+    help=INTERVAL_HELP,
+)
+@click.option(
+    "--source-endpoint",
+    required=True,
+    type=str,
+    help="ID for the source transfer endpoint",
+)
+@click.option(
+    "--dest-endpoint",
+    required=True,
+    type=str,
+    help="ID for the destination transfer endpoint",
+)
+@click.option(
+    "--label",
+    required=False,
+    type=str,
+    help=(
+        "An optional label for the transfer operation, up to 128 characters long. Must"
+        " contain only letters/numbers/spaces, and the following characters: - _ ,"
+    ),
+)
+@click.option(
+    "--stop-after-date",
+    required=False,
+    type=click.DateTime(formats=DATETIME_FORMATS),
+    help=("Stop running the transfer after this date"),
+)
+@click.option(
+    "--stop-after-runs",
+    required=False,
+    type=int,
+    help=("Stop running the transfer after this number of runs have happened"),
+)
+@click.option(
+    "--sync-level",
+    required=False,
+    type=int,
+    help=(
+        "Specify that only new or modified files should be transferred. The behavior"
+        " depends on the value of this parameter, which must be a value 0–3, as"
+        " defined in the transfer API: 0. Copy files that do not exist at the"
+        " destination. 1. Copy files if the size of the destination does not match the"
+        " size of the source. 2. Copy files if the timestamp of the destination is"
+        " older than the timestamp of the source. 3. Copy files if checksums of the"
+        " source and destination do not match."
+    ),
+)
+@click.option(
+    "--encrypt-data",
+    is_flag=True,
+    default=False,
+    help="Whether Transfer should encrypt data sent through the network using TLS",
+)
+@click.option(
+    "--verify-checksum",
+    is_flag=True,
+    default=False,
+    help=(
+        "Whether Transfer should verify file checksums and retry if the source and"
+        " destination don't match"
+    ),
+)
+@click.option(
+    "--preserve-timestamp",
+    is_flag=True,
+    default=False,
+    help=(
+        "Whether Transfer should set file timestamps on the destination to match the"
+        " origin"
+    ),
+)
+@click.option(
+    "--item",
+    "-i",
+    required=False,
+    type=(str, str, bool),
+    multiple=True,
+    help=(
+        "Used to specify the transfer items; provide as many of this option as files"
+        " to transfer. The format for this option is `--item SRC DST RECURSIVE`, where"
+        " RECURSIVE specifies, if this item is a directory, to transfer the entire"
+        " directory. For example: `--item ~/file1.txt ~/new_file1.txt false`"
+    ),
+)
```

Comment:
Is this option necessary?

I don't mean to pretend that the interface we offer in `globus endpoint transfer` is perfect, but it seems to be working well enough. We therefore need strong justification for introducing a new and different interface for an activity which is very similar and takes the same inputs.